platform/admin/ui/src/main/webapp/templates/installer/navigationButtons.handlebars

```diff
@@ -43,9 +43,11 @@
 {{else}}
     <li>
         {{#if busy}}
-            <a id="finishStep" class="disabled-next" href="#">Finish</a>
+            <a id="restartLaterStep" class="disabled-next" href="#">Restart Later</a>
+            <a id="restartNowStep" class="disabled-next" href="#">Restart Now</a>
         {{else}}
-            <a id="finishStep" class="next finish" href="#">Finish</a>
+            <a id="restartLaterStep" class="next restart-later" href="#">Restart Later</a>
+            <a id="restartNowStep" class="next restart-now" href="#">Restart Now</a>
```

Comment:
The Restart Later/Now buttons should only show if they actually changed something. If someone just hit next all the way through the installer, it should still just say Finish.